src/plugins/loadingQuotes/index.ts

```diff
@@ -99,7 +121,12 @@ export default definePlugin({
         return String.fromCharCode(...codes);
     },
 
-    get quote() {
-        return this.xor(quotes[Math.floor(Math.random() * quotes.length)]);
+    quotes(preset: string[]) {
+        const result: string[] = settings.store.enableDiscordPresetQuotes ? preset : [];
+        if (settings.store.enablePluginPresetQuotes) {
+            result.push(...quotes.map(this.xor));
+        }
+        result.push(...settings.store.additionalQuotes.split(settings.store.additionalQuotesDelimiter));
```

Comment:
split on an empty string will yield `[""]` so if you don't use the custom quote feature there's a small chance to get an empty quote
```suggestion
        result.push(...settings.store.additionalQuotes.split(settings.store.additionalQuotesDelimiter).filter(Boolean));
```